SCOPES.md

```diff
@@ -0,0 +1,70 @@
+# Prefixes
+Supported scope prefixes:
+* `project:releng:balrog`
+  * Used for Firefox/Fennec/Devedition style balrog tasks
+
+* `project:comm:thunderbird:releng:balrog`
+  * Used for Thunderbird style balrog tasks
+
+# Scopes
+* `{scope_prefix}:action:schedule`
+  * Tells balrogscript to Schedule a release to ship on balrog channel(s)
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: None
+
+* `{scope_prefix}:action:submit-locale`
+  * Tells balrogscript to submit locale information for a release.
+    * This is **default** if no `action` specified in scopes.
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: None
+
+* `{scope_prefix}:action:submit-toplevel`
+  * Tells balrogscript to submit the initial toplevel release information.
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: None
+
+* `{scope_prefix}:server:aurora`
+  * Tells balrogscript to use the aurora server config
+    * See-Also [the script config in puppet](https://dxr.mozilla.org/build-central/source/puppet/modules/balrog_scriptworker/templates/script_config.json.erb)
+    * *Informative*: This is production balrog, with aurora channel (devedition)
```

Comment:
Maybe rewrite to "This is production balrog, with aurora, aurora-localtest, and aurora-cdntest channels" to be consitent with the below one?